src/transaction/log_applier.c

```diff
@@ -4963,53 +5009,82 @@ la_apply_delete_log (LA_ITEM * item)
   if (class_obj == NULL)
     {
       assert (er_errid () != NO_ERROR);
-      error = er_errid ();
+      return er_errid ();
```

Comment:
기존 동작과 같으려면, return 전에 la_log_apply_error()를 호출해 주어야 합니다. 또는, 에러 처리를 위해 goto error; 분기를 만들고, 모든 에러 처리는 error: 레이블에서 수행하는 것도 좋을 것 같습니다.

```suggestion
      la_log_apply_error ("apply_delete", ER_HA_LA_FAILED_TO_APPLY_DELETE, item, error);
      return er_errid ();
```

Reply:
@hornetmj 리뷰 반영했습니다. 다른 함수와 마찬가지로 goto end; 추가했습니다.